src/usdb_syncer/gui/forms/SettingsDialog.ui

```diff
@@ -103,72 +106,88 @@
               </property>
              </widget>
             </item>
-            <item row="3" column="1">
-             <widget class="QCheckBox" name="checkBox_fix_first_words_capitalization">
-              <property name="text">
-               <string/>
+            <item row="1" column="1">
+             <widget class="QComboBox" name="comboBox_line_endings">
+              <property name="sizePolicy">
+               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
+                <horstretch>1</horstretch>
+                <verstretch>0</verstretch>
+               </sizepolicy>
+              </property>
+              <property name="toolTip">
+               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;USDX and variants can handle both line ending types, but if you want to edit the song file manually, your editor of choice may required a certain line ending. UNIX users (MacOS, Linux) should generally use LF (line feed). On Windows, some text editors may require CRLF (carriage return, line feed). USDB currently requires CRLF line endings.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
              </widget>
             </item>
-            <item row="4" column="0">
-             <widget class="QLabel" name="label_song_file_fix_spaces">
+            <item row="2" column="0">
+             <widget class="QLabel" name="label_song_file_format_version">
               <property name="text">
-               <string>Fix spaces:</string>
+               <string>Version:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
-             <widget class="QComboBox" name="comboBox_fix_linebreaks">
-              <property name="toolTip">
-               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If this setting is enabled, the linebreak timings will be recalculated, either according to USDX or to YASS rules.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
-              </property>
-             </widget>
+             <widget class="QComboBox" name="comboBox_format_version"/>
```

Comment:
Would be good to have an explanation what version the user should pick.

Reply:
Tried my best to add some sensible info to the tooltip.